main.go

```diff
@@ -78,6 +81,14 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
 
+	commit, err := gitCommit()
+	if err != nil {
+		setupLog.Error(err, "could not get git commit; using <undefined>")
+		commit = "<undefined>"
+	}
+
+	setupLog.Info("Creating manager", "git commit", commit)
```

Comment:
What do you think about including `vcs.modified` aka dirty?

Reply:
Why not. In fact, we could include all `vcs*` settings. WDYT?

Reply:
Yes, although `vcs` might be obvious (`git`) :)

Reply:
Nit, don't say `git commit` , "latest revision" sounds better for a log

Reply:
I like the `git commit` :)